source/includes/openings.md

```diff
@@ -29,7 +29,7 @@ close_date | <strong>integer</strong>
 
 
 ```shell
-curl https://api.recruiterbox.com/v1/openings \
+curl https://api.recruiterbox.com/api/v2/openings \
```

Comment:
We have URL rewrite rule for `api.{domain}{url}` -> `api.{domain}/public_api{url}` setup in our Nginx, eg:

`https://api.recruiterbox.com/v1/openings` gets re-written to : `https://api.recruiterbox.com/api/v2/openings`

The documentation is correct.